tests/test_barman_cloud_backup_delete.py

```diff
@@ -103,24 +103,73 @@ def _get_mock_backup_files(self, file_paths):
             for name, path in file_paths.items()
         )
 
-    def _create_backup_metadata(
-        self, backup_ids, begin_wals=None, end_wals=None, is_snapshot_backup=False
+    def _create_backup_info(
+        self,
+        backup_id,
+        begin_wals=None,
+        end_wals=None,
```

Comment:
```suggestion
        begin_wal=None,
        end_wal=None,
```

It seems a bit weird to pass the whole dict of `begin_wals` and `end_wals` to this method, if it's supposed to handle only a single backup info.

I'd say to pass only the value for that backup info, if that's the case.

Reply:
The docstring needs to be adjusted accordingly.